network/dag/bbolt_dag.go

```diff
@@ -113,7 +113,7 @@ func (dag *bboltDAG) Diagnostics() []core.DiagnosticResult {
 	})
 	result = append(result, numberOfTransactionsStatistic{numberOfTransactions: transactionNum})
 	// TODO: https://github.com/nuts-foundation/nuts-node/issues/11
```

Comment:
remove comment

Reply:
lol wonder how I managed to overlook that...